packages/dapp/src/components/Navbar/Navbar.style.tsx

```diff
@@ -305,7 +305,7 @@ export const NavbarPaper = styled(Paper, {
     '> .navbar-menu-list': {
       marginTop: 0,
       padding: !!isOpenSubMenu
-        ? openSubMenu === 'wallets'
+        ? openSubMenu === 'wallets' || openSubMenu === 'chains'
```

Comment:
Similar comment as above,
Is `openSubMenu` typed or expecting regular string?

Reply:
We had it in place: subMenuKeys - jjust gotta use it! Done now :)